### PR TITLE
Feature/book component

### PR DESCRIPTION
**Pull request by GCerbelli**
## Summary

- Add book component

## Trello Card

[Nº2.2 Componente Libro](https://trello.com/c/uBs4OBXx/10-n22-componente-libro)

## Screenshots
image

### CODE REVIEW

File: wbooks/src/app/components/BookCard/styles.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { StyleSheet } from 'react-native';
+import { Colors } from 'react-native/Libraries/NewAppScreen';
```
Comment (marianozicavo) [inline wbooks/src/app/components/BookCard/styles.ts:2]: We don't use this in current project (we could start, though).
Create a `src/constants/colors.ts` and move the colors there. Something like
```export const blue = '#288CCB';

export const transparent = 'transparent';
export const translucent = 'rgba(0,0,0,0)';

export const absoluteWhite = '#FFFFFF';
export const white = '#F9F8FD';
export const black = '#1C1C1C';

export const wildSand = '#F6F6F6';
export const whiteSmoke = '#F3F6FB';
// TODO: check if we need another constant
// export const tabooGrey = '#9B9B9B';
export const tabooGrey = '#AEADB1';
export const middleCityGrey = '#646D6D';
export const middleCityGreyOpacity = '#646D6D20';
export const nobel = '#B4B4B450';
export const spunGray = '#AEADB1';
export const spunGrayOpacity = '#AEADB120';
export const blackOpacity = `${black}CC`;
export const dimoGray = '#888888';

export const dimoBlue = '#3F20CC';

export const brandPrimary = '#2EDDC1';
export const brandPrimaryOpacity = `${brandPrimary}10`;
export const darkBrand = '#27B6A1';

export const error = '#F54632';
export const warning = '#FFCC00';
export const success = '#026F02';
```
but with the colors you'll actually use

File: wbooks/src/app/components/BookCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+
+import { styles } from './styles';
+
+const bookImage = require('./assets/img_book1.png');
```
Comment (marianozicavo) [inline wbooks/src/app/components/BookCard/index.tsx:6]:
```suggestion
import bookImage from './assets/img_book1.png';
```

Comment (GCerbelli) [reply wbooks/src/app/components/BookCard/index.tsx:6]: I didn't use import on this image cause I got thist error "Cannot find module './assets/img_book1.png' or its corresponding type declarations.ts(2307)" but I tried again, I still get this error but image loads in emulator :thinking:

File: wbooks/src/app/components/BookCard/styles.ts (hunk 1 above)
Comment (GCerbelli) [reply wbooks/src/app/components/BookCard/styles.ts:2]: Done

File: wbooks/src/app/components/BookCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+
+import { styles } from './styles';
+
```
Comment (guidoprinc) [inline wbooks/src/app/components/BookCard/index.tsx:5]: delete this empty line

File: wbooks/src/app/components/BookCard/index.tsx (hunk 2 above)
Comment (marianozicavo) [reply wbooks/src/app/components/BookCard/index.tsx]: Try adding a `index.d.ts` file in the base of the project with to see if that fixes it
`declare module '*.png';

**Pull request merged**